daemon/stats/collector.go

```diff
@@ -114,3 +115,7 @@ type notRunningErr interface {
 	error
 	ContainerIsRunning() bool
 }
+
+type notFoundErr interface {
+	error
+}
```

Comment:
Hm, looking at this again; this interface is not specific enough, as both `daemon. errNotRunning`, and `daemon. errNotFound` satisfy this interface; see https://play.golang.org/p/HtkBu1_QuM. If you make it more specific, it works though; https://play.golang.org/p/S7DzAuRITj